DigitalLearningSolutions.Web/Controllers/Support/SupportController.cs

```diff
@@ -10,6 +12,8 @@
     using Microsoft.Extensions.Configuration;
     using Microsoft.FeatureManagement;
 
+    [SetDlsSubApplication]

```

Comment:
Is this doing anything without any arguments?

Reply:
Ah yes, good question. Without any arguments the attribute will automatically try to find a route parameter of type DlsSubApplication, this is for controllers that are used in multiple applications. More info in [this documentation](https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19822084230/Selected+tab+functionality)

Reply:
Fantastic, thanks for the link!